website/integrations/services/apache-guacamole/index.mdx

```diff
@@ -21,20 +21,26 @@ The following placeholders are used in this guide:
 This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
 :::
 
-Create an OAuth2/OpenID provider with the following parameters:
+## authentik configuration
 
-- **Client Type**: `Confidential`
-- **Redirect URIs**: `https://guacamole.company/` (depending on your Tomcat setup, you might have to add `/guacamole/` if the application runs in a subfolder)
-- **Scopes**: OpenID, Email, and Profile
+To support the integration of Apache Guacamole with authentik, you need to create an application/provider pair in authentik.
 
-Under **Advanced protocol settings**, set the following:
+### Create an application and provider in authentik
 
-- **Token validity**: Any value to configure how long the session should last. Guacamole will not accept any tokens valid longer than 300 Minutes.
```

Comment:
@dominic-r did the info about the Token validity (not more than 300 minutes for Guac) get added back in?